Formula/boringtun.rb

```diff
@@ -21,12 +21,20 @@ def install
     system "cargo", "install", *std_cargo_args
```

Comment:
```suggestion
    system "cargo", "install", *std_cargo_args(path: "boringtun-cli")
```

The workspace commit after this release has broken `--HEAD` installs and requires the above change in the formula.

I am not sure whether that should be done now or later and how so figured I would point it out and leave it to you folks to decide.

Reply:
Does adding this work for 0.4.0?

Reply:
Unfortunately, no.

Reply:
The binary built is also named different: `boringtun-cli` on `HEAD` vs `boringtun` on `0.4.0`

Reply:
We are splitting boringtun up into multiple crates, so be advised, the binary name is indeed changing in `0.5.0`.

Reply:
> Unfortunately, no.

We'll have to wait to make any changes then. We'll review a pull request that uses a `build.head?` conditional to fix `--HEAD` installs in the meantime.

Reply:
yeah, I can file a separate PR to address the head build thing.

Reply:
head build fix PR, #96461